src/pages/LandingPage.tsx

```
@@ -85,25 +94,24 @@ const LandingPage: React.FC = () => {
             >
               Start Writing Now
-            </button>
-            {/* Button to track interest in the demo. */}
+            </Link>
             <button
-              onClick={() => trackUserAction('demo_request')}
+              onClick={handleWaitlistClick}
               className='btn-secondary btn-lg'
             >
-              See 30-Second Demo
+              Join Waitlist
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)